static/main.css

```diff
@@ -381,7 +381,12 @@ footer .logo {
 }
 
 #table-error {
-  color: red;
+  display: none;
```

Comment:
Other use of `#table-error` will be hidden now. We always have to change `display` when adding content to this element now. Maybe write a function in table that does it.

https://github.com/cloudamqp/lavinmq/blob/2bae422dc0ddcebf728fd7cde67f6c797e01a0d3/static/vhosts.html#L74
https://github.com/cloudamqp/lavinmq/blob/83efd006dab9cf727ce3cd44776c9216ce139d43/static/users.html#L89

Reply:
ah yes missed those, thanks!